src/accueil.c

```diff
@@ -2125,20 +2126,19 @@ gboolean gsb_main_page_update_finished_scheduled_transactions (gint scheduled_nu
     gtk_box_pack_end (GTK_BOX (hbox), label, FALSE, TRUE, 0);
     gtk_widget_show ( label);
 
-	//~ if (main_page_finished_scheduled_transactions_part)
```

Comment:
The patch I made in https://github.com/grisbi/grisbi/commit/969aef4dc6b4ea2b03d255fbcf558655e875fbaf to already fix the problem has been reverted by @pierre-biava in https://github.com/grisbi/grisbi/commit/79b9811f5845a603a31fdf37e66cdb3bee96d959.
I will let @pierre-biava decide what to do with it.